### DISCUSSION TITLE
react native gesture handler not working in android

### DISCUSSION THREAD

**Discussion by gokul1099** [software-mansion/react-native-gesture-handler#2049] [Q&A] [answered]
I am using react-native-gesture-handler in react native 0.68.  onGestureEvent in gesture handler is not getting triggered. I referred some issues in the github and they told to change the MainActivity.java as below
![image](https://user-images.githubusercontent.com/42144106/168079816-532be4aa-2de6-46ee-a6be-0f89dd3371ba.png)
I old version changing this worked, but in the newer version (0.68.2) the MainActivity.java looks different. I tried adding the above line, but while opening the it keeps on crashing without any error. here is how MainActivity.java looks like
![image](https://user-images.githubusercontent.com/42144106/168080108-9eb16c0d-9fd1-4726-af75-da881420d75c.png)
how can i fix this issue?? here is my dependencies
![image](https://user-images.githubusercontent.com/42144106/168080189-e3a8ee20-8a7d-481f-812f-d3082e06ebd8.png)

**Comment by j-piasecki** [comment, marked as answer]
The changes to `MainActivity` are not required since version 2.0.0, and `RNGestureHandlerEnabledRootView` was removed in version 2.4.0 which may be the reason for the crash (there should be a link to [the migration guide](https://docs.swmansion.com/react-native-gesture-handler/docs/guides/migrating-off-rnghenabledroot) in the stack trace). Now simply need to [wrap your app with `GestureHandlerRootView` component](https://docs.swmansion.com/react-native-gesture-handler/docs/installation#js).
